packages/core/src/exposed-thing.ts

```diff
@@ -428,7 +430,7 @@ export default class ExposedThing extends TD.Thing implements WoT.ExposedThing {
             for (const propertyName of propertyNames) {
                 // Note: currently only JSON DataSchema properties are supported
                 const form = this.properties[propertyName].forms.find(
-                    (form) => form.contentType === ContentSerdes.DEFAULT || !form.contentType
+                    (form) => form.contentType === ContentSerdes.DEFAULT || form.contentType == null
```

Comment:
```suggestion
                    (form) => form?.contentType === ContentSerdes.DEFAULT ?? true
```

Reply:
I would rather like to keep the current version 🙃

Reply:
Sure, that's fine :)